samples/wifi/provisioning/src/main.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/zephyr.h>
+
+#include "wifi_provisioning.h"
+
+void main(void)
+{
+	int rc;
+	struct wifi_config_t config;
+	struct net_if *iface = net_if_get_default();
+	struct wifi_connect_req_params cnx_params = { 0 };
+	/** Sleep 10 seconds to allow initialization of wifi driver. */
+	k_sleep(K_SECONDS(10));
+
+	rc = wifi_config_init();
+	if (rc != 0) {
+		printk("Initializing config module failed, err = %d", rc);
```

Comment:
Should this return?

Reply:
Yes. It should return if the wifi_config_init fails.